content/es/docs/reference/glossary/namespace.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Namespace
```

Comment:
```suggestion
title: Espacio de nombre (Namespace)
```
?

Reply:
Localised in https://github.com/kubernetes/website/pull/22259/commits/e423bf22516ddde02c86ee8d5e4e2f5c1c241930